api/Navigator.json

```diff
@@ -964,14 +964,28 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia",
           "support": {
-            "chrome": {
-              "version_added": "21",
-              "prefix": "webkit",
-              "notes": "Later versions of Chrome support the unprefixed <code>MediaDevices.getUserMedia()</code> which replaced this deprecated method."
-            },
-            "chrome_android": {
-              "version_added": false
-            },
+            "chrome": [
+              {
+                "version_added": "53",
+                "notes": "Unprefixed version."
```

Comment:
I think that this note is redundant.  Since there’s no “prefix” value, it’s already implied this is the non-prefixed version.